packages/amplify_authenticator/lib/src/widgets/button.dart

```diff
@@ -27,14 +25,14 @@ import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 
 extension AuthenticatorButtonSizes on AuthenticatorButtonSize {
-  double get height {
+  double? get height {
```

Comment:
Some cleanup is needed here. Most button heights were defaulting to 48px. This may be desirable for AmplifyTheme, but is much larger than the default in material. Returning null makes the buttons fallback to the material default. I left `AuthenticatorButtonSize.medium` at 40 because it is only used for social buttons, which probably do need a specific height.